### DISCUSSION TITLE
How to get multi objects in different

### DISCUSSION THREAD

**Discussion by ponytailer** [collerek/ormar#530] [Q&A] [answered]
Such like in sqlalchemy:

db.session.query(A,B).filter(A.aid == B.bid)

**Comment by collerek** [comment, marked as answer]
Right now it's not possible directly (no way to run arbitrary queries).

If A and B are related by fk/m2m you can get A as children of B or B as children of A with standard selects, otherwise you have to split into two queries - one per each model.
